src/main/java/net/minecraftforge/fml/common/IFMLSidedHandler.java

```diff
@@ -73,6 +75,9 @@
 
     IThreadListener getWorldThread(INetHandler net);
 
+    @Nullable
+    IThreadListener getMinecraftThread();
```

Comment:
This should be `getClientThread`, in my opinion. The current name suggests that it is for the thread that Minecraft is running on - be that a client or server.